src/test/java/Homework20.java

```diff
@@ -0,0 +1,51 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public class Homework20 extends BaseTest{
+
+    @Test
+    public void DeletePlaylistTest() throws InterruptedException {
+        String playlistName = "test playlist";
+
+        login("terrence.regis@gmail.com", "te$t$tudent");
+        wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//i[@title='Create a new playlist']"))).click();
+
+        WebElement newPlaylist = driver.findElement(By.xpath("//li[text()='New Playlist']"));
+        newPlaylist.click();
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.xpath("//input[@name='name']")));
+
+        WebElement nameField = driver.findElement(By.xpath("//input[@name='name']"));
+        nameField.clear();
+        nameField.sendKeys(playlistName, Keys.ENTER);
+
+        WebElement testPlaylist = wait.until(ExpectedConditions.elementToBeClickable(By.xpath("//li/a[text()='"+playlistName+"']")));
```

Comment:
- A little tip: Sometimes to do a click() action on an element you need to reach a parent element

By adding /.. or - /../{some element}

//li/a[text()='"+playlistName+"']/..